index.html

```diff
@@ -0,0 +1,188 @@
+                    </div>
+                    <div class="main-program1">
+                        <div class="main-icon">
+                            <img src="images/program_icon_03 1.svg" alt="" />
+                            <p>Forum</p>
+                        </div>
+                        <div class="main-long-write">
+                            <p>
+                                Have the time to share yout thoughts and
+                                opinions with experts for each topic
+                            </p>
+                        </div>
+                    </div>
+                    <div class="main-program1">
+                        <div class="main-icon">
+                            <img src="images/program_icon_04 1.svg" alt="" />
+                            <p>Workshop</p>
+                        </div>
+                        <div class="main-long-write">
+                            <p>
+                                Try creaing your own work using open source
+                                license rather than just watching at it.
+                            </p>
+                        </div>
+                    </div>
+                    <div class="main-program1">
+                        <div class="main-icon">
+                            <img src="images/program_icon_05 1.svg" alt="" />
+                            <p>CC Ignite</p>
+                        </div>
+                        <div class="main-long-write">
+                            <p>
+                                Get opportunities to network with CC affiliates
+                                arround the world also after the summit
+                            </p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="global-summit">
+                    <h2>Join CC Global Summit 2015</h2>
+                </div>
+                <p class="desktop-see-prog">See the whole program</p>
+            </div>
+        </section>
+
+        <section>
+            <div class="featured-speakers">
+                <h3>Featured Speakers</h3>
+                <div class="featured-line"></div>
+
+                <!-- DESKTOP SPEAKERS -->
+                <div class="desktop-speakers"></div>
+
+                <div class="more-speakers">
+                    More <img src="images/arrow_down 1.svg" alt="" />
+                </div>
+            </div>
+            <div class="first-footer">
+                <h2>Partner</h2>
+                <div class="global-line"></div>
+                <div class="footer-logos">
+                    <img src="images/mozilla 1.svg" alt="" />
+                    <img src="images/googlekorea 1.svg" alt="" />
+                    <img src="images/naver 1.svg" alt="" />
+                    <img src="images/daumkakaologo 1.svg" alt="" />
+                    <img src="images/airbnb 1.svg" alt="" />
+                </div>
```

Comment:
Great design @Developer-Kingz 🤝

- [ ] As stated in the project requirements, kindly ensure your design conforms to the design guidelines as much as possible.
- Your design
![Your partners](https://github.com/Developer-Kingz/Microverse-capstone/assets/63917725/bd8d2f21-e343-4eb2-895a-639eaab22987)
- Given design
![Given Partners](https://github.com/Developer-Kingz/Microverse-capstone/assets/63917725/717739ac-6b10-422d-ae53-7a3c80da1969)